README.md

```diff
@@ -2,47 +2,51 @@ Purge
 ------------------------------------------------------------------------------
 
 The Purge module for Drupal 8 enables invalidation of content from external
-caches, reverse proxies and CDN platforms. The technology agnostic plugin
-architecture enables different server configurations, use cases, enforces a
-separation of concerns and can be seen as **middleware** solution.
+caches, reverse proxies and CDN platforms. The technology-agnostic plugin
+architecture allows for different server configurations and use cases. Last but
+not least, it enforces a separation of concerns and can be seen as a
+**middleware** solution.
 
 Terminology overview
 ------------------------------------------------------------------------------
 
 #### Queuer
-With Purge, end users can manually invalidate a page with a Drush command or
-theoretically even with the convenience of a "clear this page"-button. Caches
-are however meant to be transparent to end users and to only invalidate when
-something actually changed - and thus requires external caches also to be
+With Purge, end users can manually invalidate a page with a Drush command or,
+theoretically, via a "clear this page" button in the GUI. Caches
+are however meant to be transparent to end users and to only be invalidated
+when something actually changed - and thus requires external caches to also be
 transparent.
 
-When editing content of any sort, Drupal will transparently invalidate cached
-pages in Drupal's own **anonymous page cache** and does this efficiently. When
-Drupal renders a page, it lists all the things on the page in a special HTTP
+When editing content of any kind, Drupal will transparently and efficiently
+invalidate cached pages in Drupal's own **anonymous page cache**. When Drupal
+renders a page, it lists all the elements on the page in a special HTTP
```

Comment:
Elements is wrong IMHO, since that suggests HTML elements but what it lists are all rendered items. So perhaps "it lists all the rendered items on the page..."